content/languages/sql/sql-writting-tests.md

```diff
@@ -0,0 +1,83 @@
+---
+description: ""
+kind: tutorial
+languages: [sql]
+topic: tests
+tags:
+  - testing
+sidebar: "language:sql"
+prev: /languages/sql/
+---
+
+# Writting tests for SQL
+
+## Generalities
+
+If you do use an existing SQL kata, be sure to avoid exposing the solution in the preloaded code file. It's easy for a code warrior to read the file in the sample tests and view the solution. Also, drop and recreate the database per `it` block so that there's no surprising state shared across cases.
```

Comment:
This is weird as a first sentence on this page.

Reply:
I'll check that one tomorrow (too much tired for that now. It's actually the original text, I just updated the post of the issue to make it compatible with the docs.

Reply:
I tried something... Can you check again, plz?